docs/_blog/bdk_cli_basics_00.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Command Line intro to Bitcoin Wallet Dev"
+description: "Intro to bdk-cli and wallet dev"
+authors:
+    - waterst0ne Pedro Felix
+date: "2022-09-22"
+tags: ["bdk-cli", "basics","novice"]
+permalink: "/_blog/bdk_cli_basics_00"
+---
+
+# Bicoin Dev Kit - Command Line Intro to Bitcoin Wallet Development 
+
+- My goal for this tutorial is to introduce you to the powerful command-line program named `bdk-cli` a  useuful tool for Bitcoin wallet testing. You will be exposed to many of the basic techincal skills that go into creating and managing Bitcoin wallets. 
+- If you've read most of the "Mastering Bitcoin" book, this tutorial could serve as a stepping stone into your Bitcoin wallet development journey.
```

Comment:
Same hyperlink love for "Mastering Bitcoin" would be nice too..

Reply:
Yes 👍